cmd/metrics-v3-cache.go

```diff
@@ -61,8 +65,15 @@ func newNodesUpDownCache() *cachevalue.Cache[nodesOnline] {
 		loadNodesUpDown)
 }
 
+type diskIOStats struct {
+	stats       map[string]madmin.DiskIOStats
+	collectedAt time.Time
```

Comment:
this can be a global var? like we have `lastDriveStatsRefresh` currently?

Reply:
I've introduced similar global vars now. This struct still stays as it's purpose is different (caching the latest drive io stats).